analyzer/repo-cloner-plugin/src/main/java/eu/fasten/analyzer/repoclonerplugin/exceptions/RepoUrlNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package eu.fasten.analyzer.repoclonerplugin.exceptions;
+
+public class RepoUrlNotFoundException extends Exception {
```

Comment:
Please add a license header and the `serialVersionUID` field.

Reply:
Resolved in [d2072f6](https://github.com/fasten-project/fasten/pull/432/commits/d2072f64d0a5431136278bb8f002f9e3ed55bca6).